### ISSUE TITLE
The accuracy of miniImagenet_ST_N5K1 experiment is 71.45%

### ISSUE THREAD

**Issue by peacelwh** [chenhaoxing/SSFormers#5]
Experiment specification of parameters:
ST.feat_dim=64
k_shot: 1
train:
    query_per_class_per_episode: 8
model:
    query: "ST"
    forward_encoding: "Grid-3,2"
test:
    query_per_class_per_episode: 8
    batch_size: 1

**Comment by peacelwh** [comment]
What's wrong? Why is accuracy so high?

**Comment by chenhaoxing** [comment]
Hi, The difference in experimental results may be from the backbone of the graphics card and pre-training models. When I use a better graphics card, I often get better results.

**Comment by peacelwh** [comment]
How big a GPU is needed for "miniImagenet_ST_N5K5_R12 experiment

**Comment by chenhaoxing** [comment]
When I was performing this experiment, I used a Tesla A100 graphics card with40 GB of memory.
